src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,38 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+class FigureSupplier {
+    public Figure getRandomFigure(int maxSize) {
```

Comment:
The method parameter 'maxSize' should be a constant in the class if it's a value that doesn't change, or it should be properly passed to the 'getRandomFigure' method. If 'maxSize' is intended to be configurable, consider defining it outside this method.